Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -129,7 +129,7 @@
   name: help me carving
   parent: BaseItem
   id: CarvingHelpMe
-  description: Help me...
+  description: Help me
```

Comment:
In my opinion there should be at least one period.

Reply:
There's actually an exclamation (according to the actual dark souls desc)